### ISSUE TITLE
Make "Reply only to sender" option visible.

### ISSUE THREAD

**Issue by thstyl2000** [nextcloud/mail#6129]
### Is your feature request related to a problem? Please describe.

At work we are often receiving emails send to a long list of recipients, and need to reply only to the sender (which is happening way more often than needing to "reply to all".

### Describe the solution you'd like

Put a "Reply only to sender" button next to the "Reply to all" one. See also #6128.

Thank you!!

**Comment by GretaD** [comment]
this is a regression, @JuliaKirschenheuter can you please add the missing action?

**Comment by ChristophWurst** [comment]
@GretaD how is that a regression?

![Bildschirmfoto von 2022-03-16 10-18-12](https://user-images.githubusercontent.com/1374172/158557278-21d629fe-5400-4a11-954f-b2888a257ae8.png)

**Comment by GretaD** [comment]
on my main branch it looks like this

![onesender](https://user-images.githubusercontent.com/12728974/158582037-8c7e0e47-9542-47eb-ab45-019545e65a25.png)

**Comment by GretaD** [comment]
ah great, i see the reply button with one arrow, i tried to reproduce on a message with one recipient and there is is not supposed to have "Send to sender only"

**Comment by thstyl2000** [comment]
Just to make sure we're talking about the same thing here: the functionality exists but there are use cases where replying to only to sender is main daily action and not replying to all.

Imagine sending a mail to ~100 recipients asking for some data (yes, universities still use email as the main data/info/collab tool...) and half of them do that by replying to all....(that's not a hypothetical scenario.....)

**Comment by ChristophWurst** [comment]
![Bildschirmfoto von 2022-03-16 15-23-06](https://user-images.githubusercontent.com/1374172/158611858-ad56217c-7a37-4abf-b6b6-e0c050af132b.png)

to take a random other email client there often is a *Reply* and *Reply all* easily accessible. For us it's one additional click to open the menu.

@nimishavijay @jancborchardt your area of expertise. Does this make sense to you? If so, what should it look like? Something like a button group?

**Comment by thstyl2000** [comment]
@ChristophWurst yes, that's what I have in mind.....thanks!

**Comment by ChristophWurst** [comment]
I'll leave the decision to you. If you want this to change then please create a mockup for the visuals.

**Comment by nimishavijay** [comment]
Something like this could work. 
![image](https://user-images.githubusercontent.com/52440189/158825293-cb202fd1-e600-446d-89e7-42b8aad745af.png)

Changes in the existing button:
- Change the text in the button to "Reply all"
- Change the icon in the button to the `reply-all` icon
We could also have a tooltip "Reply to sender only" so that it's clear.

**Comment by thstyl2000** [comment]
Sorry to bump, but wasn't sure if opening a new issue made sense:

Apparently the "reply to sender" button (when hidden in the 3dot menu) does not do anything.

**Comment by GretaD** [comment]
> Sorry to bump, but wasn't sure if opening a new issue made sense:
> 
> Apparently the "reply to sender" button (when hidden in the 3dot menu) does not do anything.

yes please, that would be very helpful to make sure this doesnt get lost. I will have a look as soon as i can
